frontend/src/Components/Profile/index.js

```diff
@@ -0,0 +1,15 @@
+import React, { useEffect, useState } from "react";
+import { Typography } from "antd";
+
+const Profile = (props) => {
```

Comment:
```
const Profile = ({ userid }) => {}
```
And you can use userid instead props.userid

Reply:
I'm not sure what you mean, when I use the following code:
`<Profile userid={userid} />`
to render this component for the current user, wouldn't the passed userid get passed inside the userid object like so?
`userid = { userid: "TtQGUtZhWNVBaXZu9BBN9lgTgvx1" }`
and I'd have to access it by calling `userid.userid'?